src/Features/LanguageServer/Protocol/AbstractRequestDispatcherFactory.cs

```diff
@@ -25,9 +25,9 @@ protected AbstractRequestDispatcherFactory(IEnumerable<Lazy<AbstractRequestHandl
         /// Creates a new request dispatcher every time to ensure handlers are not shared
         /// and cleaned up appropriately on server restart.
         /// </summary>
-        public virtual RequestDispatcher CreateRequestDispatcher(ImmutableArray<string> supportedLanguages, WellKnownLspServerKinds serverKind)
```

Comment:
We don't need the supported languages here to filter out handlers for other servers - instead we only import the handler providers that match the desired contract name